libraries/AP_Frsky_Telem/AP_Frsky_SPort_Passthrough.cpp

```diff
@@ -516,7 +516,7 @@ uint32_t AP_Frsky_SPort_Passthrough::calc_velandyaw(void)
     AP_AHRS &_ahrs = AP::ahrs();
     WITH_SEMAPHORE(_ahrs.get_semaphore());
     // horizontal velocity in dm/s (use airspeed if available and enabled - even if not used - otherwise use groundspeed)
-    const AP_Airspeed *aspeed = _ahrs.get_airspeed();
+    const AP_Airspeed *aspeed = AP::airspeed();
     if (aspeed && aspeed->enabled()) {
```

Comment:
needs explicit nullptr check instead of non-null

Reply:
It's modifying a line which isn't modified already in this PR.

I've just changed the way the pointer is fetched.  This pointer will never be nullptr more often that the existing code!

Similarly in the other telem backends.

Those are all probably wrong anyway - they should be asking the AHRS for the estimate, not the sensor.